client/app/scripts/components/terminal.js

```diff
@@ -328,4 +353,10 @@ class Terminal extends React.Component {
   }
 }
 
+
+Terminal.defaultProps = {
+  connect: true
```

Comment:
Terminal doesn't try and connect until it receives the connect attr.
`<Terminal connect />`

We don't want to connect until after the drawer animation has completed. Term.js hits the dom a lot and makes the animation super janky.